hoot-core/src/main/cpp/hoot/core/io/OgrUtilities.h

```diff
@@ -28,36 +28,57 @@
 #define OGRUTILITIES_H
 
 // GDAL
-class OGRDataSource;
+#include <gdal.h>
+// Forward declaration
+class GDALDataset;
 
 // Qt
 #include <QString>
 
 // Tgs
 #include <tgs/SharedPtr.h>
 
+#include <vector>
+
 namespace hoot
 {
 
+class OgrDriverInfo
+{
+public:
+  OgrDriverInfo(const char* indicator = NULL, const char* driverName = NULL, bool is_ext = false, unsigned int driverType = GDAL_OF_ALL)
```

Comment:
Yes I do too.  I'll do that.  Since it is just a comment and the build is quite complex with the new GDAL, I'll add it after we merge.  Same with the comment below.